src/tests/roc_pipeline/test_receiver.cpp

```diff
@@ -129,6 +129,65 @@ TEST(receiver, no_ports) {
     }
 }
 
+TEST(receiver, no_session_repair_port) {
+    config.repair_port = port1;
+
+    Receiver receiver(config, format_map, packet_pool, byte_buffer_pool,
+                      sample_buffer_pool, allocator);
+
+    CHECK(receiver.valid());
+    CHECK(receiver.add_port(port1));
+
+    PacketWriter packet_writer(receiver, rtp_composer, pcm_encoder, packet_pool,
+                               byte_buffer_pool, PayloadType, src1, port1.address);
+
+    packet_writer.write_packets(ManyPackets, SamplesPerPacket, ChMask);
+
+    FrameReader frame_reader(receiver, sample_buffer_pool);
+
+    for (size_t nf = 0; nf < ManyPackets * FramesPerPacket; nf++) {
+        frame_reader.skip_zeros(SamplesPerFrame * NumCh);
+
+        UNSIGNED_LONGS_EQUAL(0, receiver.num_sessions());
+    }
+}
+
+TEST(receiver, one_session_source_after_repair) {
```

Comment:
I suggest to rename it to one_session_drop_leading_repair_packets.

Reply:
455a825